### PR TITLE
Features/5816 combine fuzzy title description

### PR DESCRIPTION
**Pull request by utas-raymondng**
1. Update axois - due to vulnerability
2. Set inital search to bound by Tas area to speed up search
3. Pass url to user will receive the same focus area
4. Combined fuzzy_title or fuzzy_description to fuzzy_content

### CODE REVIEW

File: src/components/common/constants.js [hunk 1]
```diff
@@ -12,4 +12,11 @@ const pageDefault = {
   landing: "/",
 };
 
-export { dateDefault, pageDefault };
+const mapDefault = {
```
Comment (HavierD) [inline src/components/common/constants.js:15]: I saw you are using a same name constant in `Map.tsx`, not this one. So is it redundant?

Comment (utas-raymondng) [reply src/components/common/constants.js]: Yes you are right, fixed

**Review by HavierD — APPROVED** [review]
Looks good

File: src/components/map/mapbox/Map.tsx [hunk 2]
```diff
@@ -27,6 +27,16 @@ interface MapProps {
   ) => void;
 }
 
+const MapDefault = {
+  // Magic number, try and error by experience
+  DEBOUNCE_BEFORE_EVENT_FIRE: 1250,
+  CENTER_LONGITUDE: 147.3353554138993,
+  CENTER_LATITUDE: -42.88611707886841,
```
Comment (vietnguyengit) [inline src/components/map/mapbox/Map.tsx:34]: this coordinate is the IMAS building

Comment (utas-raymondng) [reply src/components/map/mapbox/Map.tsx:34]: Just a center point, with correct bbox it will cover most of the TAS area

**Pull request merged**